docs/UserGuide.md

```diff
@@ -261,28 +308,40 @@ Format: `find PREFIX KEYWORD [KEYWORD]…​`
 
 ### Adding a tag : `create-tag`
 
-Creates a `Tag` within WedLinker to be used on contacts.
+Use the create-tag command to create a `tag` within WedLinker, which you can assign to contacts.
 
 Format: `create-tag t/TAGNAME`
 
+To create a tag, type `tag` followed by the name of the tag.
```

Comment:
yes, though im not sure how i can add in the label part as it might make it long if i say something like 
To create a tag, type `tag` followed by the tag label `t/`  and the name of the tag.
and for commands that use multiple prefixes it might be a bit tough